### DISCUSSION TITLE
Battlesnake Error  unknown api version

### DISCUSSION THREAD

**Discussion by Snehaaa18** [BattlesnakeOfficial/feedback#254] [Q&A]
unable to create a battle snake it is showing error that it is unkown API. please help me with this issue
![Screenshot 2024-02-18 195817](https://github.com/BattlesnakeOfficial/feedback/assets/124484434/98ae4553-acfa-4518-ac86-237279eb77ec)

**Comment by paul-hansen** [comment]
The server URL you entered needs to respond with a Json object with this specification https://docs.battlesnake.com/api/webhooks#battlesnake-details
Make sure the apiversion you are returning is named correctly and the value is the string "1" (NOT a number). Any other values would cause the error you're getting.

If you need more help, if you can share the response your server URL gives when you open it in a web browser that could help us see what's happening. For example, mine responds with:
```json
{"apiversion":"1","author":"paul-hansen","color":"#bf7000","head":"do-sammy","tail":"hook","version":null}
```
